pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BatchMessageWithBatchIndexLevelTest.java

```diff
@@ -137,6 +138,67 @@ public void testBatchMessageAck() {
         });
     }
 
+    @DataProvider
+    public Object[][] enabledBatchSend() {
+        return new Object[][] {
+                {false},
```

Comment:
When it's false, even without this change, `testBatchMessageNAck` could still pass

Reply:
Yes, it only affects bacthed messages